components/Displays.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export const Displays = () => {
+  return (
+    <>
+      <div className="contentDisplay w-full bg-gray-50 border">
+        <div className="border-b bg-white w-full py-1 px-1 flex">
+          <div className="flex-1 m-auto">
+            <div className="w-6 h-6 bg-red-400 rounded"></div>
+          </div>
+          <div className="flex-1 m-auto text-center">time uploaded</div>
+          <div className="flex-1 m-auto text-right">delete</div>
+        </div>
+        <p className="p-1">Testing Iframes</p>
+        <iframe
+          className="w-full"
+          src="https://docs.google.com/presentation/d/e/2PACX-1vT21R4FsbBh7Qsf_87W-mtgvGoryzD56B6cJ6IHSvwNzSi2KWVrIlJMaxEb2ozEfZ4N7zX0-Uof3OFC/embed?start=true&loop=false&delayms=3000"
```

Comment:
This is where it should be coming from yes, but this is hardcoded. We can just use a placeholder right now, and then put whatever variable in the placeholder (that should be the path to the file) that way we can have this component displaying different files

Reply:
yeah, i think what will happen is instead of the hardcoded src, well have it pull the URL from the firestore db, so that will need to be done

Reply:
@jlbee ^

Reply:
what do you mean padding around display, theres a 2px gap between each element. Do you mean that or gap between other elements.

Reply:
Should prob be margins, and i mean around the container itself. Right now the container is the full width which goes right against the side nav, and there needs to be spacing in between the iframes as well.
![image](https://user-images.githubusercontent.com/66035845/123002927-dce5e900-d388-11eb-9913-f7b483ec2f70.png)

Reply:
just simple margin changes nothing big

Reply:
<img width="678" alt="Screen Shot 2021-06-30 at 7 02 05 PM" src="https://user-images.githubusercontent.com/49618753/123984565-c87f9d00-d9d5-11eb-976a-bc9174b26539.png">


Updated design